features/api/application_plans.feature

```diff
@@ -50,6 +50,11 @@ Feature: Application plans index page
     Then the plan is published
     And a buyer will be able to use it for their applications
 
+  Scenario: Unset the default application plan
+    When the default application plan is set
```

Comment:
Here you could set @service for later, maybe @plan 
```suggestion
    Given the service has a default application plan
```

Reply:
No need for @plan, I saved @service for later